### ISSUE TITLE
All images need to have alt text or be marked as decorative.

### ISSUE THREAD

**Issue by bedmison** [OpenDSA/OpenDSA-LTI#179]
This applies to every image in the system. The alt text should be meaningful but relatively short.

Ensure  elements have alternate text or a role of none or presentation

Fix:

To solve this problem, you need to fix at least (1) of the following:

Element does not have an alt attribute
aria-label attribute does not exist or is empty
-aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by bedmison** [comment]
List of images in app that need alt-text:

./ERD/model_report_v3/table_details.html:14:  <img src="https://raw.githubusercontent.com/OpenDSA/OpenDSA-LTI/master/ERD/OpenDSA-LTI_2021.png">
./ERD/model_report/table_details.html:14:  <img src="https://raw.githubusercontent.com/OpenDSA/OpenDSA-LTI/master/ERD/OpenDSA-LTI_v2_simple.png">
